### PR TITLE
Drupal locale set via ENV

### PR DESCRIPTION
**Pull request by tyler36**
# The Problem/Issue/Bug
Drupal defaults to English locale.

## How this PR Solves The Problem
This PR allows the user to set Drupal's locale via an ENV.

`drush si` is run with the locale being set to `DP_INSTALL_LOCALE`.
`DP_INSTALL_LOCALE`  defaults to English(`en') if not set.

No attempts are made to validate the locale code passed to drush.

## Manual Testing Instructions
Ideally, we should be able to pass the variable via the URL similar to `DP_INSTALL_PROFILE`. 
I'm sure if its that simple though.

Ideally, the browser extension would be updated to allow user to enter/select the locale. However, having it as a "undocumented" feature is fine until "full" support is required.

## Summary by CodeRabbit

- **New Features**
	- Added the ability to specify the installation profile and locale during the Drupal site setup process. Users can now set the installation locale through the `DP_INSTALL_LOCALE` environment variable, defaulting to "en" if not specified.

- **Chores**
	- Updated the Docker image reference to the latest version for improved stability and performance.

### CODE REVIEW

File: .gitpod/drupal/drupalpod-setup.sh [hunk 1]
```diff
@@ -317,7 +318,7 @@ PROJECTASYMLINK
             ddev restart
 
             # New site install
-            ddev drush si -y --account-pass=admin --site-name="DrupalPod" "$DP_INSTALL_PROFILE"
+            ddev drush si -y --account-pass=admin --site-name="DrupalPod" "$DP_INSTALL_PROFILE" --locale="{$DP_INSTALL_LOCALE:=en}"
```
[the remark this answers is not in the document]
Comment (tyler36) [reply .gitpod/drupal/drupalpod-setup.sh:321]: The suggestion is invalid if run in container.

```shell
drush si -y --account-pass=admin --site-name="DrupalPod" "$DP_INSTALL_PROFILE" --locale="{$DP_INSTALL_LOCALE:-en}"
...
  Translation:The <em class="placeholder">-en</em> translation is not available.<br /><br />The <em class="placeholder">-en</em> translation file is not available at the translation server. <a href="/index.php">Choose a differ  
  ent language</a> or select English and translate your website later.  
```

[the remark this answers is not in the document]
Comment (tyler36) [reply .gitpod/drupal/drupalpod-setup.sh:321]: The original runs correctly in the container but there seems to be an issue when DDEV passes it through.

```
$ ddev drush si -y --account-pass=admin --site-name="DrupalPod" --locale={$DP_INSTALL_LOCALE:=en}
...

 Translation:The <em class="placeholder">jaen</em> translation is not available.<br /><br />The <em class="placeholder">jaen</em> translation file is not available at the translation server. <a href="/index.php">Choose a different language</a> or select English and translate your website later.

[the remark this answers is not in the document]
Comment (tyler36) [reply .gitpod/drupal/drupalpod-setup.sh]: Yes, thank you. ID10-T error. You win this round AI, well done!

File: .gitpod/drupal/drupalpod-setup.sh [hunk 2]
```diff
@@ -16,6 +16,7 @@ if [ -n "$GITPOD_HEADLESS" ]; then
     export DP_EXTRA_DEVEL=1
     export DP_EXTRA_ADMIN_TOOLBAR=1
     export DP_PROJECT_TYPE='default_drupalpod'
+    export DP_INSTALL_PROFILE=en
```
[the remark this answers is not in the document]
Comment (tyler36) [reply .gitpod/drupal/drupalpod-setup.sh:19]: We use `DP_INSTALL_PROFILE` to set the Drupal installation profile. Eg. "standard", "demo_umami".
`DP_INSTALL_LOCALE` is used to explicitly set the install language in the case where the installtion profile is not used or is used but not explicitly set.

**Comment by tyler36** [comment]
Reset / rebase on main

File: .gitpod/drupal/drupalpod-setup/drupalpod-setup.sh [hunk 3]
```diff
@@ -136,8 +137,7 @@ GITMODULESEND
     # ddev config auto updates settings.php and generates settings.ddev.php
     ddev config --auto
     # New site install
-    time ddev drush si -y --account-pass=admin --site-name="DrupalPod" "$DP_INSTALL_PROFILE"
-
+    time ddev drush si -y --account-pass=admin --site-name="DrupalPod" "$DP_INSTALL_PROFILE" --locale="${DP_INSTALL_LOCALE:-en}"
```
[the remark this answers is not in the document]
Comment (tyler36) [reply .gitpod/drupal/drupalpod-setup/drupalpod-setup.sh:140]: Yes, please assist in documenting the DP_INSTALL_LOCALE variable usage in the project's README or developer documentation

**Comment by rfay** [comment]
But this repo has moved to drupal.org right? Shouldn't an MR be there? 

@shaal you can easily do a mirroring setup with github actions, but you have to decide where the authoritative code is.

**Comment by shaal** [comment]
Thank you @rfay
The official code is hosted in d.o - https://git.drupalcode.org/project/drupalpod

**Comment by tyler36** [comment]
Closing.

Will submit MR on https://git.drupalcode.org/project/drupalpod

**Pull request closed without merging**